### PR TITLE
[OPENJDK-1812] correct jre module version for ubi9/openjdk-17-runtime

### PR DESCRIPTION
**Pull request by jmtd**
https://issues.redhat.com/browse/OPENJDK-1812

I broke this image with the Maven 3.8 update PR. This should fix it.

### CODE REVIEW

**Comment by sefroberg** [comment]
Looks good to me.

File: ubi9-openjdk-17-runtime.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ modules:
   install:
   - name: jboss.container.util.pkg-update
   - name: jboss.container.openjdk.jre
-    version: "3.8.17"
+    version: "17"
```
Comment (jerboaa) [inline ubi9-openjdk-17-runtime.yaml:44]: Makes sense. Only the maven modules (not installed for runtime), have version scheme `<maven-version>.<jdk-version>`. Nonsensical for module `jboss.container.openjdk.jre`. Thumbs up.

**Review by jerboaa — APPROVED** [review]
I had to take a look at https://github.com/jboss-container-images/openjdk/pull/338 and https://github.com/jboss-container-images/openjdk/pull/347 again to understand what has been done here. [This](https://github.com/jboss-container-images/openjdk/pull/338/files#diff-457f9e6063c66ef1981a6b5aa3ba59879f4f2f9095592d388d77900769118600R44) was the breaking change.

**Comment by jmtd** [comment]
> I had to take a look at https://github.com/jboss-container-images/openjdk/pull/338 and https://github.com/jboss-container-images/openjdk/pull/347 again to understand what has been done here. [This](https://github.com/jboss-container-images/openjdk/pull/338/files#diff-457f9e6063c66ef1981a6b5aa3ba59879f4f2f9095592d388d77900769118600R44) was the breaking change.

Sorry I'll include more context in future.

**Pull request merged**